src/main/java/org/folio/circulation/domain/notice/schedule/DueDateNotRealTimeScheduledNoticeHandler.java

```diff
@@ -33,24 +37,29 @@ public static DueDateNotRealTimeScheduledNoticeHandler using(Clients clients, Da
       DueDateScheduledNoticeHandler.using(clients, systemTime),
       new LoanRepository(clients),
       new LoanPolicyRepository(clients),
-      PatronNoticeService.using(clients));
+      PatronNoticeService.using(clients),
+      TemplateRepository.using(clients));
   }
 
   private final DueDateScheduledNoticeHandler dueDateScheduledNoticeHandler;
   private final LoanRepository loanRepository;
   private final LoanPolicyRepository loanPolicyRepository;
   private final PatronNoticeService patronNoticeService;
+  private final TemplateRepository templateRepository;
+  private static final String TEMPLATE_RECORD_TYPE = "template";
```

Comment:
Personally, I would define constants at the very top of the class, followed by private fields.